opentracing/tracer.go

```diff
@@ -0,0 +1,71 @@
+package opentracing
+
+import (
+	basictracer "github.com/opentracing/basictracer-go"
+	opentracing "github.com/opentracing/opentracing-go"
+	"sourcegraph.com/sourcegraph/appdash"
+)
+
+var _ opentracing.Tracer = NewTracer(nil) // Compile time check.
+
+type Options struct {
+	// ShouldSample is a function that allows deterministic sampling of a trace
+	// using the randomly generated Trace ID. The decision is made when a new Trace
+	// is created and is propagated to all of the trace's spans. For example,
+	//
+	//   func(traceID int64) { return traceID % 128 == 0 }
+	//
+	// samples 1 in every 128 traces, approximately.
+	ShouldSample func(traceID int64) bool
```

Comment:
Should document that Appdash does not support sampling currently, and will ignore this field.